job-service-db/src/main/resources/procedures/dropTables.sql

```diff
@@ -0,0 +1,46 @@
+--
+-- Copyright 2016-2021 Micro Focus or one of its affiliates.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+/*
+ *  Name: drop_tables
+ *
+ *  Description:
+ *  Drops all tables present in delete_log table
+ *  This procedure does batch commits. The batch is defined by commit_limit variable. Default batch size being 10.
+ */
+
+CREATE OR REPLACE PROCEDURE drop_tables()
```

Comment:
Can we call this `drop_deleted_task_tables`?  `drop_tables` seems very generic.

Reply:
I kept it generic so that the same piece of code can be reused in the future. But I am fine changing it if you think this needs to be specifically named.

Reply:
Yes rename it.

Reply:
Renamed.